src/preparers.py

```diff
@@ -454,6 +454,16 @@ def prepare_ultrachat(row):
             parent = i
         return messages
 
+def prepare_wildchat(row):
+    messages = []
+    for i, script_dict in enumerate(row['conversation']):
+        messages.append({
+            'from': script_dict['role'],
+            'text': script_dict['content'].strip(),
+            'parent': row['model'] if i==0 else 0
```

Comment:
Shouldn't it be `else i` rather than `else 0`? What if the conversation is longer than 2 turns.

`i` is the 0-index of the conversation turn that this one is responding to.